spikeinterface/sorters/runsorter.py

```diff
@@ -174,7 +188,7 @@ def __init__(self, mode, container_image, volumes, extra_kwargs):
             options=['--bind', singularity_bind]
 
             # gpu options
-            if extra_kwargs.get('requires_gpu', False):
+            if gpu_to_use:
                 # only nvidia at the moment
                 options += ['--nv']
```

Comment:
Is it possible to specify which gpu to use also for singularity?

Reply:
This should be a separate PR, this functionality is not in the package yet (as far as I can see).